sdk/boms/azure-sdk-bom/README.md

```diff
@@ -30,8 +32,8 @@ result in all dependencies being included in your project.
 
 ### Adding libraries to your project
 
-After adding the BOM all artifacts included in the BOM are now available to be added as a dependency without listing the
-artifact's version.
+After adding the BOM Azure client libraries included in the BOM are now available to be added as a dependency without 
```

Comment:
"After adding the BOM, Azure..." there should be a comma after the prepositional phrase.

Reply:
Added the `,`.